src/Integration.Vsix/source.extension.vsixmanifest

```diff
@@ -1,17 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="b039a29c-3109-44e1-9695-cededc24dfec" Version="0.9" Language="en-US" Publisher="SonarSource SA and Microsoft Corporation" />
+    <Identity Id="SonarLint.8c442ec8-0208-4840-b83f-acd24f41acf7" Version="2.0.0" Language="en-US" Publisher="SonarSource SA and Microsoft Corporation" />
     <DisplayName>SonarLint for Visual Studio</DisplayName>
-    <Description xml:space="preserve">Connect Visual Studio and SonarQube to enforce code development rules and comply with governance policies, and to minimize your technical debt</Description>
-    <MoreInfo>https://github.com/SonarSource-VisualStudio/sonarlint-visualstudio</MoreInfo>
+    <Description xml:space="preserve">Provides on-the-fly feedback to developers on new bugs and quality issues injected into C# and VB.Net code. Further, it allows connecting Visual Studio and SonarQube to enforce shared code development rules and comply with governance policies, and to minimize your technical debt.</Description>
+    <MoreInfo>http://vs.sonarlint.org</MoreInfo>
     <License>License.txt</License>
     <GettingStartedGuide>https://github.com/SonarSource-VisualStudio/sonarlint-visualstudio/blob/master/README.MD</GettingStartedGuide>
     <Icon>Resources\sonarqube_32.png</Icon>
     <PreviewImage>Resources\sonarqube_200.png</PreviewImage>
+    <Tags>SonarLint, SonarQube, Analysis, Roslyn, CodeAnalysis, Analyzer, Code analysis, Sonar, Debt, Technical, Tech, Quality</Tags>
   </Metadata>
   <Installation>
     <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[14.0]" />
+    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[14.0]" />
```

Comment:
I'm not sure that these are needed or not as we could already install the VSIX into Enterprise editions. However we had them previously, and I'm unsure if the visual studio gallery requires them or not to show the proper version support.